docs/modules/ROOT/pages/mgmt_services/route_manager.adoc

```diff
@@ -37,24 +37,24 @@ image::route_manager_service_architecture.PNG[]
 
 === Key Workflow
 
-** For case #4, add or delete a non-gateway port.
+==== For case #4, add or delete a non-gateway port.
 
 image::route_manager_l3_neighbors_update_4.PNG[]
```

Comment:
We have a minor change in implementation for workflow #4. Instead of returning a list of subnet ids, we return a list of SubnetEntity.

Reply:
@cj-chung Bring this up for your attention.

Reply:
I have changed workflow #4 to include Subnet_Entities in the return body.